### DISCUSSION TITLE
How to add composite unique condition

### DISCUSSION THREAD

**Discussion by felstormrage** [payloadcms/payload#1946] [Q&A] [answered]
How can I add a composite unique condition on two fields (author, slug)

``` 
{
  name: 'slug',
  type: 'text',
},
{
  name: 'author',
  type: 'relationship',
  relationTo: 'creators',
},
```

**Comment by JarrodMFlesch** [comment, marked as answer]
@rednithin you could make a third field that concatenates these two (with a hook), and mark that field as unique.
